src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -241,6 +241,13 @@ internal void ComputeReturnType()
                 // Note: we don't need to warn on annotations used without NonNullTypes context for local functions, as this is handled in binding already
             }
 
+            // span-like types are returnable in general
+            if (returnType.IsRestrictedType(ignoreSpanLikeTypes: true))
```

Comment:
>ignoreSpanLikeTypes: true [](start = 44, length = 25)

It would be good to have a tests observing significance of ```ignoreSpanLikeTypes: true```.

Reply:
Should it be a standalone test in LocalFunctionTests.cs instead of something in SymbolErrorTests.cs?

Reply:
Addressed in 640f73e